dc_utils/templates/dc_base.html

```diff
@@ -44,35 +43,32 @@
                 </main>
             {% endblock main_base %}
 
+
+            {% if USE_I18N %}
```

Comment:
I'm not sure this block is needed? I think Django sets `LANGUAGE_CODE` automaticlly if the `i18n` context processor is enabled. If it's not, the below if statement will fall back to the `else` condition.

Can you check this?

Reply:
You're right, I've removed that block.